ROADMAP.md

```diff
@@ -0,0 +1,65 @@
+
+## Goals
+
+- Short-term Goals: Focus on GPT-4 / Claude-3 or API-based projects.
+- Demo Goal: Aim to replicate existing demos.
+- Accuracy Goal: Work on the evaluation of the SWE-Bench (aiming for a 15% accuracy improvement) and Dev-Bench.
+- Long-term Goal: Explore the potential for automatic coding/software engineering advancements.
+
+## Milestones
+
+- TODO
+
+## Roadmap
+
+1. Community operation
+   1. Establish an open source committee and clarify the board members and management mechanism. This committee is mainly used for SOP design and operation.
+   2. Set up maintainer(s) and join some developers into the github group. They can determine which code can be incorporated and push everything forward.
+   3. Design and provide community operation SOP documents, determine entry and exit criteria, and determine the responsibilities of different roles.
+2. Use Case and interaction
+   1. Document learning: Supports specifying a URL or keyword to learn documents, use corresponding documents, and use tools.
+   2. Repair based on issue: Given a github repo and issue, the Agent can issue a Pull Request for Review
+   3. Confidential information mosaic: supports inquiry and mosaic of confidential information (such as passwords)
```

Comment:
I wonder if we can integrate with 1password or other password managers, rather than giving credentials directly to OpenDevin.

This seems like maybe a stretch feature--probably not one we want to hack in, and not critical for short-term goals